update tld support section (#48)
AdguardTeam/KnowledgeBase: docs/general/ad-filtering/create-own-filters.md

### Wildcard support for TLD (top-level domains) {#wildcard-for-tld}

Wildcard characters are supported for TLDs of the domains in patterns of [cosmetic](#cosmetic-rules), [HTML filtering](#html-filtering-rules) and [javascript](#javascript-rules) rules.

> For example, the cosmetic rule `example.*##.banner` will match any ICANN-registered TLD request, i.e. `example.com`, `example.net`, `example.org`, etc.

Reviewer: не, тут как раз не `ICANN-registered TLD`, а public suffixes - можно на него ссылку дать https://publicsuffix.org/list/

Reviewer 2: In addition we should mention AdGuard for iOS and Safari here. There we use a shorter list of top-level-domains instead.

> For example, in the cosmetic rule `example.*##.banner` the part `.*` will match any [public suffix](https://publicsuffix.org/list/), i.e. `example.com`, `example.net`, `example.org`, etc.

Reviewer: я тут ещё подумал, что можно упомянуть термин eTLD (effective top-level domain, по сути синоним public suffix в данном контексте)

ну и мне кажется, что сейчас фраза не совсем корректно построена, может как-то так?
```suggestion
For example, in the cosmetic rule `example.*##.banner` the part `.*` matches any [public suffix (or eTLD)](https://publicsuffix.org/list/), i.e. the rule matches `example.com`, `sub.example.net`, `example.co.uk`, etc.
```

### Wildcard support for TLD (top-level domains) {#wildcard-for-tld}

Wildcard characters are supported for TLDs of the domains in patterns of [cosmetic](#cosmetic-rules), [HTML filtering](#html-filtering-rules) and [javascript](#javascript-rules) rules.

> For cosmetic rules, e.g. `example.*##.banner`, multiple domains will be matched due to the part `.*`, i.e. `example.com`, `sub.example.net`, `example.co.uk`, etc.
> 
> For basic rules the described logic will be applicable only for the domains specified in [`$domain` modifier](#domain-modifier), e.g. `||*/banners/*$image,domain=example.*`.

Reviewer: по идее лучше без will тут
```suggestion
For cosmetic rules, e.g. `example.*##.banner`, multiple domains are matched due to the part `.*`, i.e. `example.com`, `sub.example.net`, `example.co.uk`, etc.

For basic rules the described logic is applicable only for the domains specified in [`$domain` modifier](#domain-modifier), e.g. `||*/banners/*$image,domain=example.*`.
```
